AWSCoreTests/AWSTestUtility.m

```diff
@@ -65,6 +65,57 @@ + (void)initialize {
     [super initialize];
 }
 
++ (void) setupSessionCredentialsProvider {
+    
+    NSDictionary *testConfigurationJson = [self getTestConfigurationJSON];
+    AWSBasicSessionCredentialsProvider *credentialsProvider = [[AWSBasicSessionCredentialsProvider alloc]
+                                                               initWithAccessKey:testConfigurationJson[@"Credentials"][@"accessKey"]
+                                                                       secretKey:testConfigurationJson[@"Credentials"][@"secretKey"]
+                                                                    sessionToken:testConfigurationJson[@"Credentials"][@"sessionToken"]];
+    AWSServiceConfiguration *configuration = [[AWSServiceConfiguration alloc] initWithRegion: [testConfigurationJson[@"Credentials"][@"region"] aws_regionTypeValue]
```

Comment:
It seems weird that "region" would be in the "credentials" block of the configuration.

Reply:
Aiming to have a single region for all test resources. Handling multiple regions needs to be discussed.

Reply:
"region" is shifted to "common" block which also has a default cognito identity pool and related info to be used by tests requiring authorization using cognito idp (cognito should be supported in that region)